CRM/Export/BAO/Export.php

```diff
@@ -2118,6 +2117,8 @@ public static function setHeaderRows($field, $headerRows, $sqlColumns, $query, $
       $headerRows[] = $field;
     }
 
+    self::manipulateHeaderRows($headerRows, $contactRelationshipTypes);
```

Comment:
QA'd the PR, works fine and corrects the header for relationships. Just thinking if we could avoid calling of this method for every field and instead move this call to a little upward immediately after [`$headerRows` of contact relationships are evaluated](https://github.com/civicrm/civicrm-core/blob/master/CRM/Export/BAO/Export.php#L2118) ?